ballerina/api_cv_attachment_srv/modules/mock/api_cv_attachment_srv_mock_service.bal

```diff
@@ -0,0 +1,64 @@
+// AUTO-GENERATED FILE.
+// This file is auto-generated by the Ballerina OpenAPI tool.
+
+import ballerina/http;
+// import ballerina/os;
+
+// boolean isBalBuild = os:getEnv("IS_BAL_BUILD") == "true";
```

Comment:
This needs to be reverted

Reply:
Updated in [Update review changes](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/11/commits/efc0871542242b1e5b587b00abe2fd99465c9fbe)